plugins/com.google.cloud.tools.eclipse.appengine.libraries.test/src/com/google/cloud/tools/eclipse/appengine/libraries/PomTest.java

```diff
@@ -70,6 +76,15 @@ public void setUp() throws SAXException, IOException, CoreException {
     pomFile.create(in, IFile.FORCE, null);
    
     pom = Pom.parse(pomFile);
+    
+    Logger logger = Logger.getLogger(ArtifactRetriever.class.getName());
+    logger.setLevel(Level.OFF);
+  }
+  
+  @After
+  public void tearDown() throws SAXException, IOException, CoreException {
+    Logger logger = Logger.getLogger(ArtifactRetriever.class.getName());
+    logger.setLevel(Level.WARNING);
```

Comment:
Should use `logger.setLevel(null)` so that it inherits from the parent (which ws probably the previous level)?